### ISSUE TITLE
fix: link and buttons contrast ratio

### ISSUE THREAD

**Issue by ramboz** [hlxsites/petplace#153]
The current "white/salmon" color combo (`#ffffff`/`#ff7d5a`) is not offering enough contrast to meet the WCAG guidelines.
How do we want to proceed here?

See also: https://adobe-dx-support.slack.com/archives/C04P5DFPDK9/p1682368204865649
